slam-libraries/viam-cartographer/src/slam_service/slam_service.cc

```diff
@@ -0,0 +1,112 @@
+#include "slam_service.h"
+
+#include <iostream>
+#include <string>
+
+#include "../mapping/map_builder.h"
+#include "cartographer/mapping/map_builder.h"
+#include "glog/logging.h"
+#include "server_functions.h"
+
+namespace viam {
+// GetPosition returns the relative position of the robot w.r.t the "origin" of
+// the map, which is the starting point from where the map was initially
+// created.
+::grpc::Status SLAMServiceImpl::GetPosition(ServerContext *context,
+                                            const GetPositionRequest *request,
+                                            GetPositionResponse *response) {
+    std::cout << "GetPosition" << std::endl;
+    return grpc::Status::OK;
```

Comment:
[nit] I would make this error that it is unimplemented.

Reply:
Sorry, I meant, can you make this return an error, like here: https://github.com/viamrobotics/slam/blob/457f1c3cd997b10e3f3c262a00a21d0f87a3d70d/slam-libraries/viam-orb-slam3/orbslam_server_v1.cc#L88?

Reply:
Yeah I just realized 😅

Reply:
Maybe this? `return grpc::Status(grpc::StatusCode::UNIMPLEMENTED);`

Reply:
Yes, looks good!